services/web-server/src/entities/Artifact.js

```diff
@@ -3,7 +3,8 @@ module.exports = class Artifact {
     Object.assign(this, data);
     this.taskId = taskId;
     this.isPublic = /^public\//.test(this.name);
-    this.isLog = /^text\/plain*/.test(this.contentType);
```

Comment:
I wasn't sure if this `/^text\/plain*/` was a bug -- that means we can have 0 to infinite `n` characters, matching `text/plain` or `text/plainnnnnnnn`, no? Intentional?

Reply:
That was probably not intentional :)